.circleci/config.yml

```diff
@@ -1341,12 +1401,19 @@ workflows:
             - cleanup-rerun-testnet-if-present
           workflow-name: "Continuous integration"
       - cleanup-singlejob-testnet:
+          context: Slack
           requires:
             - run-smart-contract-scenarios
           pre-steps:
             - attach_workspace:
                 at: /
 jobs:
+  test-slack:
+    executor: ci-test-executor
+    steps:
+      - checkout
+      - svc-app-slack-msg: 
+          text: Testing new API token from CircleCI context env
```

Comment:
This job should be removed before merging to master.

Reply:
Done.